services/QuillLMS/app/models/student_activity_sequence.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: student_activity_sequences
+#
+#  id                        :bigint           not null, primary key
+#  created_at                :datetime         not null
+#  updated_at                :datetime         not null
+#  classroom_id              :integer          not null
+#  initial_activity_id       :integer          not null
+#  initial_classroom_unit_id :integer          not null
+#  user_id                   :integer          not null
+#
+class StudentActivitySequence < ApplicationRecord
+  belongs_to :classroom
```

Comment:
The [RFC](https://www.notion.so/quill/RFC-for-Pre-Post-data-storage-ade408cd92724dec9807be31bd3be12b?pvs=4#7a08786044864950a246d72fdac7f2f0) listed
`student_id`
`classroom_id` 
`initial_activity_id`

But now there is a `initial_classroom_unit_id`.  A couple of questions,
1) Do we need classroom if we now have classroom_unit?
2) Why is this called `initial_classroom_unit`?  Could there be a change in the course of the student activity sequence?  (If not, I would probably just call it `classroom_unit`)

Reply:
Thank you for that explanation.  I misunderstood the data model here, so I think `initial_classroom_unit` makes sense.

As far as `classroom_id` I'll be curious to hear what others think but I think your de-normalization argument for reporting seems reasonable.